site/docs/pages/index.mdx

```diff
@@ -62,27 +86,86 @@ bun add @coinbase/onchainkit
 
       </div>
       <a href="/getting-started" className="font-sans justify-self-end border rounded-xl border-solid border-white px-4 py-3 font-bold">
-        View docs 
+        View docs
       </a>
     </div>
 
   </div>
 </div>
+<div className="flex justify-between items-center md:mb-[64px] mx-auto max-w-screen-xl">
+  <div className="text-[40px] dark:text-white font-normal w-[365px] leading-[48px] tracking-[-1.6px]">
+    <p>
+      Ready-to-use onchain components
+    </p>
+  </div>
+  <div className="flex gap-12">
+    <div className="flex flex-col items-center gap-3">
```

Comment:
I feel like all of these should be a UL / LI
so that the SEO can better understand we are showcasing a list.